src/common/xo/index.js

```diff
@@ -1205,6 +1205,12 @@ export const disconnectVif = vif =>
 
 export const deleteVif = vif => _call('vif.delete', { id: resolveId(vif) })
 
+export const deleteVifs = vifs =>
+  confirm({
+    title: _('deleteVifsModalTitle', { nVifs: vifs.length }),
+    body: _('deleteVifsModalMessage', { nVifs: vifs.length }),
+  }).then(() => map(vifs, vif => _call('vif.delete', { id: resolveId(vif) })))
```

Comment:
Add `noop` as `.then`'s 2nd parameter so that anything using the `confirm`'s promise wouldn't catch an error when the user closes the modal.